### ISSUE TITLE
Some text does not meet WCAG standards for accessible text.

### ISSUE THREAD

**Issue by HonkingGoose** [tailwindlabs/heroicons.com#10]
## Summary of problem:

The Firefox accessibility toolkit flags certain text as too low in contrast.

## Where is the problem?

Low text color contrast in the following areas:

- A lot of de-emphasized text in the grape colored hero section.
- The light gray text used right after the Medium and Small `<h2>` tags.
- In the footer, the Twitter blue text

## Proposed solution

From the [Mozilla MDN on Color contrast](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast):
> When choosing a color scheme for your website, choose foreground and background colors that have good contrast. Make the color contrast as good as it can be within your design constraints — ideally go for the AAA rating (see 1.4.6 below), but at least meet the AA rating (see 1.4.3 below).

The main way that I solve this problem for my designs, is to darken the background when I'm using light text colors. And to darken the color on white/gray backgrounds.

## Screenshot of flagged issues:

![low-contrast-text](https://user-images.githubusercontent.com/34918129/91345852-4e442200-e7e0-11ea-86ea-91d26660b2fb.png)

## References to learn more:

The accessibility inspector is explained here: https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector

Mozilla MDN guidelines for accessible text colors here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast

WCAG standard for text color: https://www.w3.org/TR/WCAG21/#contrast-minimum

**Comment by quick007** [comment]
I'm not gonna lie, this is basically the twitter color, I can't see them needing to change this 
![image](https://user-images.githubusercontent.com/69743171/135566812-50a0fa27-67af-4cfb-b523-3f2bf60cd2eb.png)

**Comment by adamwathan** [comment]
Hey thanks! While we work very hard to make sure things are keyboard accessible and work properly with assistive technology (we have projects like [Headless UI](https://headlessui.dev/) for example that are purpose-built to solve those problems), color contrast is trickier because you have to make real trade-offs in the design, and visual design is really important to us as well. In this case, we've made sure the vast majority of the page has sufficient contrast, but for little decorative things that only really exist to look nice (like the "2px stroke weight, 24x24 bounding box" text), meeting WCAG contrast guidelines isn't a goal. Not going to make any changes right now, but appreciate it regardless.

_Worth noting that the WCAG is moving to a new color contrast formula for 3.0, which is being developed here and is a lot more forgiving for white text on colored backgrounds than the flawed methodology that they have used up until now: https://github.com/Myndex/SAPC-APCA_
